haystack/modeling/data_handler/processor.py

```diff
@@ -381,7 +381,7 @@ def __init__(
         doc_stride: int = 128,
         max_query_length: int = 64,
         proxies: Optional[dict] = None,
-        max_answers: int = 6,
+        max_answers: int = None,
```

Comment:
We need to adapt the type here:
```suggestion
        max_answers: Optional[int] = None,
```